### DISCUSSION TITLE
Action event "schedule" does not support all cron options

### DISCUSSION THREAD

**Discussion by dz0ny** [community/community#24874] [Apps, API and Webhooks]
<p>The following cron value (“30 6 7-14 * 1”) <a href="https://crontab.guru/#30_6_7-14_*_1" rel="nofollow noopener">https://crontab.guru/#30_6_7-14_*_1</a> is rejected by the workflow, while “30 6 * * 1” is accepted. Looks like range is not supported?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b37026f10699980200311e15d93f8d23c968e0ab.png" data-download-href="/uploads/short-url/pBnKCIPPibZmQwF1XvyFVlcmD7J.png?dl=1" title="Screenshot-20190724112808-1490x409.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/239513/181087150-b8cba4f7-a54f-4b6c-a6eb-691c842f3628.png" alt="Screenshot-20190724112808-1490x409.png" data-base62-sha1="pBnKCIPPibZmQwF1XvyFVlcmD7J" width="690" height="189" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b37026f10699980200311e15d93f8d23c968e0ab_2_690x189.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/b37026f10699980200311e15d93f8d23c968e0ab_2_1035x283.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/b37026f10699980200311e15d93f8d23c968e0ab_2_1380x378.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b37026f10699980200311e15d93f8d23c968e0ab_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot-20190724112808-1490x409.png</span><span class="informations">1490×409 32.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for letting us know. Ranges are stated as being specifically supported, so I’m not sure that’s the root cause of the problem. In any case, I’ve let the developers know about the issue. I can’t give any promises of when or even if it will be addressed, but I’ve reported it to the proper people.</p>
<p>Thanks again for reaching out!</p>
